src/test/java/nl/tudelft/pixelperfect/event/EventParametersTest.java

```diff
@@ -111,6 +122,14 @@ public void testEqualsDifferentSizes() {
   }
 
   /**
+   * An EventParameters instance should no be considered equal to something of another type.
```

Comment:
Not in my opinion. Tests at this level are just to meet coverage guidelines, they hardly test any functionality. Nevertheless, I wil make an additional test case for an instance of another class.